src/main/scala/PrivateHome/data.scala

```diff
@@ -53,20 +55,21 @@ object data {
   def create(dropTables: Boolean = false): Unit = {
 
     // Drops old Tables to Delete Data and make it possible to regenerate them
-    if (dropTables)
-    sql"""DROP TABLE IF EXISTS `Mhz`;
+    if (dropTables) {
+      sql"""DROP TABLE IF EXISTS `Mhz`;
          DROP TABLE IF EXISTS `Devices`;
          DROP TABLE IF EXISTS `User`""".execute().apply()
+      devices = mutable.Map()
+    }
 
 
-    val devices =
       sql"""
                     CREATE TABLE IF NOT EXISTS `Devices` (
                     `id` varchar(5) NOT NULL,
                     `name` varchar(64) NOT NULL,
-                    `type` varchar(16) NOT NULL,
+                    `switchtype` varchar(16) NOT NULL,
                     `state` decimal(5,4) NOT NULL,
-                    `keepState` boolean NOT NULL,
+                    `keepstate` boolean NOT NULL,
```

Comment:
WHY?!!! Why do you changed away from camelCase?

Reply:
Simply because it gave weird errors were the higher level API made it to something like keep_state so this was the easiest way.